### ISSUE TITLE
Add alt-text to images in templates

### ISSUE THREAD

**Issue by kimberscott** [lookit/lookit-api#715]
See https://github.mit.edu/Accessibility-FY2021/lookit/issues/11. NSF logo is missing alt text; check for any others on templates.
